lib/gitsh/environment.rb

```diff
@@ -88,7 +87,7 @@ def repo_has_untracked_files?
     end
 
     def repo_config_color(name, default)
-      if color_override = self[name]
+      if color_override = fetch(name) { nil }
```

Comment:
I haven't seen this syntax with `fetch` before - what is the purpose of passing `{ nil }` here?